cns/fsnotify/fsnotify.go

```diff
@@ -56,17 +57,19 @@ func (w *watcher) releaseAll(ctx context.Context) {
 	defer w.lock.Unlock()
 	for containerID := range w.pendingDelete {
 		// read file contents
-		filepath := w.path + "/" + containerID
+		filepath := filepath.Join(w.path, containerID)
 		file, err := os.Open(filepath)
 		if err != nil {
 			w.log.Error("failed to open file", zap.Error(err))
```

Comment:
As a reader of the logs, do I have enough information here to troubleshoot? Keep in mind the reader might be not familiar with the code or be technical.